app/Http/Requests/BlogPostRequest.php

```diff
@@ -24,9 +24,9 @@ public function authorize()
     public function rules()
     {
         return [
-            'username' =>'required|alpha',
-            'email' =>'required|email|unique:users',
-            'password' => 'required| min:5| max:10'
+            'user_id' => 'required',
```

Comment:
we can also validate if user_id exists in database or not 
`['user_id' => 'exists:users,id']`